### PR TITLE
W-11131351 Fix dependabot issues

### PR DESCRIPTION
**Pull request by spelak-salesforce**
GUS [W-11131351](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000wrGebYAE/view)

Fixes dependabot issues:
- [Prototype Pollution in yargs-parser #1](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/1)
- [Regular Expression Denial of Service in trim-newlines #2](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/2)
- [Inefficient Regular Expression Complexity in nth-check #3](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/3)
- [Exposure of sensitive information in follow-redirects #4](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/4)
- [Exposure of Sensitive Information to an Unauthorized Actor in follow-redirects #6](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/6)
- [Prototype Pollution in minimist #8](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/8)

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
Refer to [Definition of Done](https://salesforce.quip.com/9P7hAOPHJJyU) to see any additional details for the items below:
- [x] Any net new LWC work has JEST test coverage 50% or above
- [x] Default Sa11y tests pass for all LWC components
- [x] 🔒 Secure both Front-end (LWC) & back-end (Apex) as necessary
- [x] 🔑 Grant users access in Permission Sets (Object, Field, Apex Class) as necessary
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: [W-0000000: Work Name]()
- [x] Make sure that ACs are updated (if any gaps)
- [x] **All acceptance criteria have been met**
    - [x] Developer
    - [x] Code Reviewer
- [x] Pull Request contains draft release notes
- [x] Labels, help text, and customer facing messages are reviewed by Docs
- [x] QE story level testing completed

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7786,22 +6724,7 @@ yaml@^1.10.0, yaml@^1.10.2, yaml@^1.7.2:
...
-  version "18.1.3"
-  resolved "https://registry.yarnpkg.com/yargs-parser/-/yargs-parser-18.1.3.tgz#be68c4975c6b2abf469236b0c870362fab09a7b0"
-  integrity sha512-o50j0JeToy/4K6OZcaQmW6lyXXKhq7csREXcDwk2omFPJEwUNOVtJKvmDr9EI1fAJZUyZcRF7kxGBWmRXudrCQ==
-  dependencies:
-    camelcase "^5.0.0"
-    decamelize "^1.2.0"
-
-yargs-parser@^20.2.2:
+yargs-parser@^10.0.0, yargs-parser@^18.1.2, yargs-parser@^20.2.2, yargs-parser@^21.0.0:
   version "20.2.9"
```
Comment (spelak-salesforce) [inline yarn.lock:6728]: Fixes dependabot issue [Prototype Pollution in yargs-parser #1](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/1).

File: yarn.lock [hunk 2]
```diff
@@ -5803,13 +5205,6 @@ npm-run-path@^4.0.0, npm-run-path@^4.0.1:
   dependencies:
     path-key "^3.0.0"
 
-nth-check@^1.0.2:
```
Comment (spelak-salesforce) [inline yarn.lock:5806]: Fixes dependabot issue [Inefficient Regular Expression Complexity in nth-check #3](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/3) as nth-check is no longer used and removed from `yarn.lock`.  Do a find for `nth-check@` to verify it's not used.

File: yarn.lock [hunk 3]
```diff
@@ -3651,14 +3118,14 @@ flat-cache@^3.0.4:
...
-  integrity sha512-8/sOawo8tJ4QOBX8YlQBMxL8+RLZfxMQOif9o0KUKTNTjMYElWPE0r/m5VNFxTRd0NSw8qSy8dajrwX4RYI1Hw==
+  version "3.2.5"
+  resolved "https://registry.yarnpkg.com/flatted/-/flatted-3.2.5.tgz#76c8584f4fc843db64702a6bd04ab7a8bd666da3"
+  integrity sha512-WIWGi2L3DyTUvUrwRKgGi9TwxQMUEqPOPQBVi71R96jZXJdFskXEmf54BoZaS1kknGODoIGASGEzBUYdyMCBJg==
 
 follow-redirects@^1.14.0:
-  version "1.14.5"
-  resolved "https://registry.yarnpkg.com/follow-redirects/-/follow-redirects-1.14.5.tgz#f09a5848981d3c772b5392309778523f8d85c381"
-  integrity sha512-wtphSXy7d4/OR+MvIFbCVBDzZ5520qV8XfPklSN5QtxuMUJZ+b0Wnst1e1lCDocfzuCkHqj8k0FpZqO+UIaKNA==
+  version "1.15.0"
```
Comment (spelak-salesforce) [inline yarn.lock:3126]: Fixes dependabot issues [Exposure of sensitive information in follow-redirects #4](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/4) and [IExposure of Sensitive Information to an Unauthorized Actor in follow-redirects #6](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/6).

File: yarn.lock [hunk 4]
```diff
@@ -5668,10 +5087,10 @@ minimist-options@^3.0.1:
     arrify "^1.0.1"
     is-plain-obj "^1.1.0"
 
-minimist@^1.1.1, minimist@^1.2.0, minimist@^1.2.5:
-  version "1.2.5"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.5.tgz#67d66014b66a6a8aaa0c083c5fd58df4e4e97602"
-  integrity sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==
+minimist@^1.1.1, minimist@^1.2.0, minimist@^1.2.5, minimist@^1.2.6:
+  version "1.2.6"
```
Comment (spelak-salesforce) [inline yarn.lock:5091]: Fixes dependabot issue [Prototype Pollution in minimist #8](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/8).

**Review by mldyang — APPROVED** [review]

**Comment by mldyang** [comment]
Hi @davidjray this PR is ready for RE review. Can you please take a look when you get the chance? Thank you.

**Review by davidjray — APPROVED** [review]

**Pull request merged**